x/liquidity/client/cli/query.go

```diff
@@ -216,12 +216,12 @@ func GetCmdQueryPoolBatchDepositMsgs() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "deposits [pool-id]",
 		Args:  cobra.ExactArgs(1),
-		Short: "Query for all deposit messages on the batch of the liquidity pool specified pool-id",
+		Short: "Query all deposit messages of the liquidity pool batch",
 		Long: strings.TrimSpace(
-			fmt.Sprintf(`Query for all deposit messages on the batch of the liquidity pool specified pool-id
+			fmt.Sprintf(`Query all deposit messages of the liquidity pool batch on the specified pool
 
-if batch messages are normally processed and from the endblock,  
-the resulting state is applied and removed the messages from the beginblock in the next block.
+if batch messages are normally processed from the endblock,
+the resulting state is applied and removes the messages from the beginblock in the next block.
```

Comment:
Logically, i think this should be `the resulting state is applied, and the messages are removed in the beginning of next block.`